pkg/service/test/test.go

```diff
@@ -257,6 +271,16 @@ func (t *tester) Test(path string, testReportPath string, appCmd string, options
 			break
 		}
 	}
+
+	pp.Printf("\n <=========================================> \n  COMPLETE TESTRUN SUMMARY. \n\tTotal tests: %s\n"+"\tTotal test passed: %s\n"+"\tTotal test failed: %s\n", totalTests, totalTestPassed, totalTestFailed)
+
+	pp.Printf("\n\tTest Suite Name\t\tTotal Test\tPassed\t\tFailed\t\n")
+	for testSuiteName, verdict := range completeTestReport {
+		pp.Printf("\n\t%s\t\t%s\t\t%s\t\t%s", testSuiteName, verdict.Third, verdict.First, verdict.Second)
+	}
+
```

Comment:
print the result of test suites in sorted order

Reply:
done